float32 environment_temperature
PixelTuple min
PixelTuple max
PixelTuple spot